### ISSUE TITLE
Add "CITATION.cff" file and asign Zenodo DOI

### ISSUE THREAD

**Issue by yvonnefroehlich** [GenericMappingTools/agu24workshop#38]
Looking at the EGU 2022 workshop, I am wondering if we also want to add a `CITATION.cff ` file to the repo and asign a Zenodo DOI to make them the materials citeable.

**Remember to update the AGU URLs!!!**

Related EGU 2022 workshop PR: https://github.com/GenericMappingTools/egu22pygmt/pull/20

**Comment by weiji14** [comment]
Yes, I'll add this to my TODO list, will do it sometime in 2025, once all the other PRs have been merged.
